packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md

```diff
@@ -114,6 +114,18 @@ The following table provides a list of all available email templates in an Okta
 | Reassigned Review | Access certification campaign: (`campaignName`) | `IGAReviewerReassigned` | | Sent to reviewers when they are assigned new reviews by an admin or a different reviewer. |
 | Idp MyAccount Email Change Confirmation | Confirm email address change | `MyAccountChangeConfirmation` |  | Sent to users who try to verify an email address using MyAccount APIs. The users must enter the provided code to confirm the change. |
 
+### Use Branding variables
+
+After you create a theme for your org, select the `FULL_THEME` ("Solid Background") variant to use branding variables in your custom email templates.
+
+> **Note:** If you delete any of the following optional variables from the email code, you disconnect the theme object properties from the email template.
+
+| Variable | Asset Type |
+|-------------------|--------------------|
+| `${brand.theme.logo}`| URL |
+| `${brand.theme.primaryColor}` | Hex code for the CTA button |
+| `${brand.theme.secondaryColor}` | Hex code for the background color |
```

Comment:
Would it be better if we clarified the specific format of the hex code - e.g., rgb, rgba?

Reply:
in the format of this guide, the user doesn't need to know the format, the color will just magically populate, so they don't need to add a value or anything. In the brands API reference, they do need to know the format and I believe we cover it there